### ISSUE TITLE
OCR Windows 10 broken on Windows version  10.0.16299.15?

### ISSUE THREAD

**Issue by dreinn** [nvaccess/nvda#7659]
Pressing NVDA-r, it say start recognizing and then, failed.
If I go to NVDA preference, ocr, I can't select any language.
I have an italian pc, with English and italian installed languages.

ERROR - contentRecog.recogUi._recogOnResult (11:13:31.062):
Recognition failed: UWP OCR initialization failed

**Comment by feerrenrut** [comment]
Could you please do the following, and attach the log file to this issue. See [this wiki article](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) if you are unsure how to do this.
* Set your logging level to "Debug"
* Save config and restart NVDA
* Open NVDA preferences, OCR
* Try to recognise something with `NVDA+r`
* Attach the full log to this issue.

**Comment by ivnc** [comment]
A Spanish-speaking user has contacted nvda.es by Twitter with apparently the same error, and here is the relevant log fragment: [core_7659-M_Rivera-3-relevant_part.log](https://github.com/nvaccess/nvda/files/2316443/core_7659-M_Rivera-3-relevant_part.log)
Do you need the full debug log? Have we to open OCR settings dialog before pressing NVDA+R? In this log he (the user) didn't it, but if it's necessary the problem could be reproduced again including that step.

Regards.

**Comment by Adriani90** [comment]
@ivnc, @dreinn I was not able to reproduce this issue with NVDA 2019.1.1 on Windows 10 1809 update. Are you still having this issue or have you recently hear any additional user reports on this?

**Comment by ivnc** [comment]
Hi.

I’ve just got in touch with the user again to know if he’s still experiencing this issue. I’ll let you know as soon as I get a response from him.

Thanks. Regards.

**Comment by ivnc** [comment]
The user in question haven't replied yet, but another one as complained about a similar problem. He has checked to have the Spanish (spain) OCR installed and there are no languages in the NVDA combo box to select the preferred one. I'll post the log as soon as I receive it.

Regards.

**Comment by Adriani90** [comment]
@ivnc did you get any log files from the users?

**Comment by ivnc** [comment]
No, the user in question didn't reply to my last message (sent in May 2019) and we haven't received any more reports on this.
I suggest closing this issue. Anyway, if there are any updates (I've just sent another message to the user) I'll let you know.

Regards.

**Comment by ivnc** [comment]
sorry, I forgot: with regard to the second user, currently I didn't remember who they was and the status of their case, but I think that they solved it by installing (or re-installing) the corresponding regional Windows pack.

Regards.

**Comment by Adriani90** [comment]
Thanks Iván for reporting back. Closing as works for me.
If the issue stil persists we can reopen it later.
